src/cmd_line.c

```diff
@@ -8,15 +8,273 @@ See the file LICENSE for details.
 #include "console.h"
 #include "string.h"
 #include "testing.h"
+#include "iso.h"
 
 #define KEYBOARD_BUFFER_SIZE 256
+#define ROOT "/"
 
-//Privates
+char cur_path[256]; // The current working directory for the command line environment
+
+void move_up_directory(char *abs_path);
```

Comment:
Do we want these functions directly in the cmd_line module? I think it makes more sense for them to live in a filesystem_commands module, and keep cmd_line pure